src/rust/graph-merger/src/config.rs

```diff
@@ -0,0 +1,6 @@
+#[derive(clap::Parser, Debug)]
+#[clap(name = "graph-merger", about = "Graph Merger Service")]
+pub struct GraphMergerConfig {
+    #[clap(env = "GRAPH_MUTATION_CLIENT_URL")]
+    pub graph_mutation_client_url: String,
+}
```

Comment:
Nope. Use the pre-existing `src/rust/rust-proto/src/client_factory/services/graph_mutation.rs` like all existing code:

```
    let mutation_client_config = GraphMutationClientConfig::parse();
    let mut graph_mutation_client = build_grpc_client(mutation_client_config).await?;
```

Reply:
I fixed this